includes/core-changes/networking/2.0/behavior-change-webrequest-deprecate.md

```diff
@@ -0,0 +1,34 @@
+### WebRequest is not recomended for new development
+
+Starting in .NET Core 2.0, <xref:System.Net.WebRequest>, <xref:System.Net.WebClient> and <xref:System.Net.ServicePoint> support was added to .NET Core in version 2.0 for backward compatibility. The reason for returning the API is for API backquard compatibility, however there are considerable runtime breaking changes.
+
+#### Change description
+
+For example:
+
+ 1. WebRequest.GetRequestStream alocate memory for whole response
+ 2. WebClient.CancelAsync doesn't always cancel immediately
+
+#### Version introduced
+
+2.0
+
+#### Recommended action
+
+Use the <xref:System.Net.Http.HttpClient?displayProperty=fullName> class instead of <xref:System.Net.WebClient?displayProperty=fullName>, which is deprecated.
```

Comment:
Is WebClient deprecated in .NET Core 2.0 or .NET 5.0?

This file mentions that WebClient is deprecated in .NET Core 2.0, while the other file in the PR mentions that WebRequest, WebClient, and ServicePoint are all deprecated in .NET 5.0.

That confuses me a little bit.

Reply:
There are two messages:
1) We don't recommend use WebRequest api (.net core 2.0 when the classes were added)
2) Add Obsolete attribute to WebRequest api (.net 5.0)

Reply:
I honestly see the breaking change in 5.0 only.

How "not recommending an API without obsoletion" is considered a breaking change?

Am I missing something?

Reply:
I agree with Youssef here. I think we should remove this file and the link where it's included.